### PR TITLE
Homing logic fix

### PR DESCRIPTION
**Pull request by Badulundabad**
Now seeker controller is using target from targeted crosshair

### CODE REVIEW

File: CSharpSourceCode/Abilities/Ability.cs [hunk 1]
```diff
@@ -297,7 +297,16 @@ private void AddBehaviour(ref GameEntity entity, Agent casterAgent)
                 }
                 else
                 {
-                    AbilityScript.SetTargetSeeking(new Target {Formation = casterAgent.Formation.QuerySystem.ClosestSignificantlyLargeEnemyFormation.Formation}, Template.SeekerParameters); //TODO: Need crosshair
+                    Target target;
+                    if (Crosshair.CrosshairType == CrosshairType.Targeted)
+                    {
+                        target = new Target { Agent = (Crosshair as TargetedCrosshair).Target };
```
Comment (hunharibo) [inline CSharpSourceCode/Abilities/Ability.cs:303]: what happens when there is no current target on the crosshair object? we should prevent activation of ability then.

**Review by hunharibo — COMMENTED** [review]
One question, otherwise no comments. Good stuff.

File: CSharpSourceCode/Abilities/Ability.cs (hunk 1 above)
Comment (Badulundabad) [reply CSharpSourceCode/Abilities/Ability.cs:303]: Yeah, this check is already here, in AbilityManagerMissionLogic, line 156

**Review by hunharibo — APPROVED** [review]
Approved

**Pull request merged**